projects/components/src/draggable-list/draggable-list.component.scss

```diff
@@ -9,3 +9,21 @@
     cursor: not-allowed;
   }
 }
+
+.draggable-list.cdk-drop-list-dragging .draggable-item:not(.cdk-drag-placeholder) {
```

Comment:
these cdk classes aren't ours - can we better control them - they shouldn't be stylable like this with encapsulation

Reply:
Can I try to style it from outside but I should use ng-deep

Reply:
I don't think these would work without ng deep, but ideally we're not using that either. Why do we need to change the out of the box animation? And the preview they give us explicit controls over if we want to customize it via `cdkDragPreviewClass`

Reply:
I'm adding this styles to animate the dragging effect. If I add it cdkDragPreviewClass I will need to add it in the parent class of <ng-container> so I'm not having the chance to add it from outside.

Reply:
Got it - so I assumed some of this was default behavior, but it looks like the docs examlpe are doing this in the parent too (guessing that's the inspiration here)

Reply:
I will keep it for now.